docs/ref/contrib/contenttypes.txt

```diff
@@ -344,6 +344,22 @@ model:
    objects. See :ref:`natural keys<topics-serialization-natural-keys>` and
    :option:`dumpdata --natural-foreign` for more information.
 
+.. admonition:: Behavior change in Django 5.1
+
+   In Django 5.1, model forms were not able to declare a form field for a
+   GenericForeignKey field anymore, raising a FieldError ``cannot be specified
+   for %s model form as it is a non editable field``. In the next release, the
+   old behavior has been restored. To restore the old behavior in Django 5.1,
+   it's possible to monkey patch the field with ``editable=True``:
+
+   .. code-block:: python
+
+        class TaggedItem(models.Model):
+            content_object = GenericForeignKey("content_type", "object_id")
+
+
+        TaggedItem.editable = True
+
```

Comment:
As we will fix this and back port onto Django 5.1, we don't need to document here :+1: 
Instead we need a little release note in the Django 5.1.6 release to let folks know we've fixed this, something like:
```diff
--- a/docs/releases/5.1.6.txt
+++ b/docs/releases/5.1.6.txt
@@ -12,3 +12,6 @@ Bugfixes
 * Fixed a regression in Django 5.1.5 that caused ``validate_ipv6_address()``
   and ``validate_ipv46_address()`` to crash when handling non-string values
   (:ticket:`36098`).
+
+* Fixed a regression in Django 5.1 that where ``GenericForeignKey`` fields
+  could not be used in a :class:`~django.forms.ModelForm` (:ticket:`36151`).
```